test/models/test-users.js

```diff
@@ -0,0 +1,15 @@
+const { expect } = require('chai');
+const sinon = require('sinon');
+const db = require('../../src/models');
+
+const { User } = db;
+
+describe('User functionality', () => {
+  it('should be a model object', () => {
+    sinon.stub(User.model, 'findAll');
+    sinon.stub(User.model, 'findOne');
+    sinon.stub(User.model, 'update');
+
+    expect(User.model);
```

Comment:
What even is this asserting?

Reply:
This is starting out as a generic test boilerplate in the boilerplate - lol

Reply:
Still a work in progress...:P